README.md

```diff
@@ -41,39 +41,41 @@ This tool expects to run in an environment with the following attributes:
 
 Requesting a code review:
 
-    git appraise request
+    git review request
 
 Pushing code reviews to a remote:
 
-    git appraise push [<remote>]
+    git review push [<remote>]
 
 Pulling code reviews from a remote:
 
-    git appraise pull [<remote>]
+    git review pull [<remote>]
 
 Listing open code reviews:
 
-    git appraise list
+    git review
+    git review --list
 
 Showing the status of the current review, including comments:
 
-    git appraise show
+    git review show
 
 Showing the diff of a review:
 
-    git appraise show --diff [--diff-opts "<diff-options>"] [<review-hash>]
+    git review show -p/-u/--patch [--diff-opts "<diff-options>"] [<review-hash>]
 
 Commenting on a review:
 
-    git appraise comment -m "<message>" [-f <file> [-l <line>]] [<review-hash>]
+    git review comment -m "<message>" [-f <file> [-l <line>]] [<review-hash>]
 
 Accepting the changes in a review:
 
-    git appraise accept [-m "<message>"] [<review-hash>]
+    git review accept [-m "<message>"] [<review-hash>]
 
 Submitting the current review:
 
-    git appraise submit [--merge | --rebase]
+    git review merge [--ff | --no-ff | --ff-only ]
+    git review rebase
```

Comment:
In this case, the name was chosen a bit haphazardly; I don't think it ever occurred to me to use the names of the existing git concepts for submitting a review. In particular, there are (as @hazbo pointed out) semantic differences between submitting and merging/rebasing/fast-forwarding, so reusing the existing git terminology could cause some confusion.

A good example for what could cause issues is the "rebase" option. I've actually been thinking of adding a "git appraise rebase" command that would not submit the review. This would help teams that want to force each review to be a single commit (e.g. the most common workflow for Gerrit). That potential command would just kick off an interactive rebase, and then copy over the review data to point at the resulting new commit.

Similarly, you could imagine a "git appraise merge" command that merges the current review onto the head of the target ref without submitting it (but no one has asked for such a thing).

The reason I went with "submit" was that it is a commonly used term for getting changes into a codebase (mostly in centralized source control systems like Subversion and Perforce), and that felt like a closer analog to what "git appraise submit" does than merge/rebase/etc.